src/App.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import '../src/css/style.css';
-import Navbar from './pages/Navbar';
+import '../src/css/responsive.css';
+import Navbar from './components/Navbar';
+import CreateAccount from './components/CreateAccount';
 import LoginPage from './components/LoginPage';
+import ProfilePage from './pages/ProfilePage/ProfilePage';
 import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom';
 
+
 function App() {
     return (
         <Router>
           {/* <Navbar /> */}
           <Routes>
-            <Route exact path="/userlogin" element={<LoginPage />} />
+            <Route exact path="/" element={<LoginPage />} />
```

Comment:
Why we have two route for login page

Reply:
I missed to remove the second route of login page. Doing it now !